resources/functionMap_php80delta.php

```diff
@@ -102,7 +102,7 @@
 		'str_ends_with' => ['bool', 'haystack'=>'string', 'needle'=>'string'],
 		'str_starts_with' => ['bool', 'haystack'=>'string', 'needle'=>'string'],
 		'strchr' => ['string|false', 'haystack'=>'string', 'needle'=>'string', 'before_needle='=>'bool'],
-		'stripos' => ['int|false', 'haystack'=>'string', 'needle'=>'string', 'offset='=>'int'],
+		'stripos' => ['0|positive-int|false', 'haystack'=>'string', 'needle'=>'string', 'offset='=>'int'],
```

Comment:
changed to have the same return type across php versions, so we don't need to ignore different errors per php version